src/Constants.js

```diff
@@ -90,7 +90,7 @@ exports.GiveawayStartOptions = {};
  *
  * @param {Discord.GuildMember} member
  * @param {Giveaway} giveaway
- * @returns {Promise<number>|number}
+ * @returns {Discord.Awaitable<number>}
```

Comment:
```suggestion
 * @returns {Promise<number>|number}
```
i think it's better to leave it be, cause on the website it will appear just like that.
someone would have to look up what "awaitable" means.